spec/lib/hca/enrollment_system_spec.rb

```diff
@@ -1554,6 +1554,10 @@
       [
         'application/octet-stream',
         'PDF'
+      ],
+      [
+        'application/unknown-mime',
+        'PDF'
```

Comment:
Is our intention for this that if the mime type is not found in our allowable types that we set it as a `PDF`? Are there concerns with sending an invalid file to the ES as a `PDF`?